index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" />
+    <title>My Portfolio</title>
+</head>
+<body>
+    <header class="header">
+        <a class="logo" href="#">Rubanza Mark</a>
+        <nav>
+          <a class="menu" href="#"
+            ><img src="img/IconMenu.png" alt="IconMenu"
+          /></a>
+        </nav>
+      </header>
```

Comment:
Kindly add some padding-top for the `header` section, to be like the Figma design.

Your Design
![image](https://user-images.githubusercontent.com/35535130/153926003-737561ed-d347-46c1-8902-ed93e8a02b37.png)

Reply:
All good now 👍🏽